packages/react-aria-components/docs/CheckboxGroup.mdx

```diff
@@ -366,7 +367,11 @@ function MyCheckboxGroup({
   );
 }
 
-function MyCheckbox({children, ...props}: CheckboxProps) {
+interface MyCheckboxProps extends Omit<CheckboxProps, 'children'> {
+  children?: React.ReactNode
```

Comment:
I think this is to replace RenderProps definition of children? seems like we would need to do className and style as well?

Reply:
No because style and className actually get passed to the Checkbox (where function is supported), but children is passed along with other stuff (the function isn't the only child).